gems/aws-sdk-core/lib/aws-sdk-core.rb

```diff
@@ -54,9 +54,12 @@ def self.crt_loaded?
   end
 end
 
+# SigV4
+require_relative 'aws-sdk-core/auth_schemes/sigv4'
+require_relative 'aws-sdk-core/credentials'
```

Comment:
This seems off .. Will there be other credential types? Sigv4a technically should be its own credential type? Second guessing this naming. Possibly this should remain in identities/ for future AWS identity (sigv5?)

Reply:
Follow up - identities/credentials IMO